packages/xchain-litecoin/src/utils.ts

```diff
@@ -120,7 +120,7 @@ export const getBalance = async (params: AddressParams): Promise<Balance[]> => {
       },
     ]
   } catch (error) {
-    return Promise.reject(new Error('Invalid address'))
+    throw new Error('Invalid address')
```

Comment:
```ts
Error('Could not get balances for address ${params.address}') // ...not sure what the address is in `AddressParams`
```

Reply:
yep, it's `.address`
```suggestion
    throw new Error(`Could not get balances for address ${params.address}`)
```